charts/refarch-templates/templates/hpa.yaml

```
@@ -1,6 +1,6 @@
 {{- $dot := . }}
-{{- range $module := .Values.modules -}}
-{{ $data := dict "dot" $dot "module" $module }}
+{{- range $moduleName, $module := .Values.modules -}}
+{{ $data := dict "dot" $dot "module" $module "moduleName" $moduleName }}
```

YAMLlint (1.38.0)

[error] too many spaces inside braces

(braces)